pages/pipelines/defining_steps.md.erb

```diff
@@ -270,6 +270,24 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+## Dynamic pipeline templates
+
+If you need the ability to use pipelines from a central catalog, or enforce certain configuration rules, you can either use dynamic pipelines and the [`pipeline upload`](/docs/agent/v3/cli-pipeline#uploading-pipelines) command to make this happen, or [write custom plugins](/docs/plugins/writing) and share them across your organization.
```

Comment:
```suggestion
If you need the ability to use pipelines from a central catalog, or enforce certain configuration rules, you can either use dynamic pipelines and the [`pipeline upload`](/docs/agent/v3/cli-pipeline#uploading-pipelines) command to make this happen or [write custom plugins](/docs/plugins/writing) and share them across your organization.
```

Reply:
Only tiiiiny comments in this review! :) It feels like many commas already, and this one could definitely be omitted.